### ISSUE TITLE
replace oracle jdk with openjdk

### ISSUE THREAD

**Issue by csantanapr** [ibm-functions/runtime-kotlin#3]
Currently using oracle
```
RUN curl --silent --location --retry 3 --cacert /etc/ssl/certs/GeoTrust_Global_CA.pem \
  --header "Cookie: oraclelicense=accept-securebackup-cookie;" \
  http://download.oracle.com/otn-pub/java/jdk/"${VERSION}"u"${UPDATE}"-b"${BUILD}"/d54c1d3a095b4ff2b6607d096fa80163/server-jre-"${VERSION}"u"${UPDATE}"-linux-x64.tar.gz \
  | tar xz -C /tmp && \
```

We should replace and use the same one we are using in runtime-java
```
FROM adoptopenjdk/openjdk8-openj9:jdk8u162-b12_openj9-0.8.0
```
https://github.com/apache/incubator-openwhisk-runtime-java/blob/master/java8/Dockerfile#L17

**Comment by csantanapr** [comment]
@seabaylea can you take a look at this one?

**Comment by csantanapr** [comment]
Closed by #8
